tests/db/sqlalchemy/conftest.py

```diff
@@ -0,0 +1,108 @@
+import pytest
+import os
+import dotenv
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy import create_engine
+from datetime import datetime
+from harvester.db.models import Base
+
+dotenv.load_dotenv()
+
+
+@pytest.fixture(scope="session")
+def engine():
+    host = os.getenv("POSTGRES_HOST")
+    user = os.getenv("POSTGRES_USER")
+    password = os.getenv("POSTGRES_PASSWORD")
+    db = os.getenv("POSTGRES_DB")
+    dialect = "psycopg"
+
+    engine = create_engine(f"postgresql+{dialect}://{user}:{password}@{host}/{db}")
+    yield engine
+    engine.dispose()
+
+
+@pytest.fixture(scope="session")
+def tables(engine):
+    Base.metadata.create_all(engine)
+    yield
+    Base.metadata.drop_all(engine)
+
+
+@pytest.fixture(scope="session")
+def session(engine, tables):
+    Session = sessionmaker(bind=engine)
+    session = Session()
+    yield session
+    session.rollback()
+    session.close()
+
+
+@pytest.fixture
+def test_harvest_source():
+    return {
+        "id": "efecc0d6-9dac-4916-8356-150a683e9864",
+        "name": "test_harvest_source",
+        "notification_emails": ["something@example.com", "another@test.com"],
+        "organization_name": "example",
+        "frequency": "weekly",
+        "config": {"test": 1, "example": True, "something": ["a", "b"]},
+        "urls": [
+            "https://path_to_resource/example.json",
+            "https://path_to_resource/another_example.json",
+        ],
+        "schema_validation_type": "dcatus",
+    }
+
+
+@pytest.fixture
+def test_harvest_job():
+    return {
+        "id": "c8837a2f-bd0b-4333-82f5-0b64f126da34",
+        "source_id": "efecc0d6-9dac-4916-8356-150a683e9864",
+        "status": "INACTIVE",
+    }
```

Comment:
`status` here does not match the ones defined in [`job_status` enum](https://github.com/GSA/datagov-harvesting-logic/pull/15/files#diff-c85ae75006b1235644ceaec61ce07d6f4e3369520e1370750617695bda0180d2R78-R82)

Reply:
updated [here](https://github.com/GSA/datagov-harvesting-logic/pull/15/commits/b76dee49da957addb3773fdf5903881f650e75e7)